README.md

```diff
@@ -111,98 +110,6 @@ steps:
 
 [embedmd]:# (tmp/help.txt)
```

Comment:
Hmm... can't say I've ever seen this before?

Reply:
From my understanding this is used to embed snippets into other markdown files as part of Drone CI steps. However, I believe this may only have been used for the old (deprecated) method of pushing documentation to Drone CI plugins. They changed this and now pull READMEs of Drone Plugins vs. allowing plugins to push.

See [here](https://github.com/drone/drone-plugin-index/blob/main/plugins/cache/content.yaml)